ppr-ui/src/views/amendment/ConfirmAmendment.vue

```diff
@@ -628,23 +548,11 @@ export default defineComponent({
         return
       }
 
-      // get registration data from api and load into store
-      await loadRegistration()
-
-      // page is ready to view
-      emitHaveData(true)
-      localState.dataLoaded = true
-    }
-
-    /** Emits Have Data event. */
-    const emitHaveData = (haveData: Boolean = true): void => {
-      context.emit('haveData', haveData)
-    }
-
-    /** Emits error to app.vue for handling */
-    const emitError = (error: ErrorIF): void => {
-      context.emit('error', error)
-      console.error(error)
+      if (!localState.registrationNumber || !getConfirmDebtorName.value ||
+          localState.registrationNumber !== getRegistrationNumber.value) {
```

Comment:
I added any extra check to make sure that state is matching the registration, otherwise it will return the user to dashboard. This could occur if a user tries to go to review screen directly by typing in the URL instead of going through the amendment page first.